tools_docker/Debian_Uwsgi/common.sh

```diff
@@ -19,6 +19,10 @@ if [ ! -f /etc/tracim/development.ini ]; then
     sed -i "s|webdav.listen = .*|webdav.listen = 127.0.0.1:3030|g" /etc/tracim/development.ini
     sed -i "s|;webdav.root_path = /|webdav.root_path = /webdav|g" /etc/tracim/development.ini
     sed -i "s|webdav.base_url = .*|webdav.base_url = http://localhost:8080|g" /etc/tracim/development.ini
+    sed -i "s|email.processing_mode = sync|email.processing_mode = async|g" /etc/tracim/development.ini
+    sed -i "s|; email.async.redis.host = .*|email.async.redis.host = localhost|g" /etc/tracim/development.ini
+    sed -i "s|; email.async.redis.port = .*|email.async.redis.port = 6379|g" /etc/tracim/development.ini
+    sed -i "s|; email.async.redis.db = .*|email.async.redis.db = 0|g" /etc/tracim/development.ini
```

Comment:
what is "0" for `email.async.redis.db` ? its the name of the database?

Reply:
I don't know :(
Its the default value in development.ini.sample (and in code) (I don't have take time to know what is this parameter).

@inkhey can you explain here this parameter?

Reply:
Actually, this is the value that sounds strange (if its the db name I would have expected something like tracim_cache or similar... but maybe it's redis which only use numbers for naming databases/indexes?)

Reply:
This paramter seem to be used if you used multiple redis database. But by default the config (in documentation) is `null`. I don't know if `null` and `0` is the same for redis.

Reply:
Redis used number and not name for database.

Reply:
ok